_posts/research/2020-04-22-robotic-skin.md

```diff
@@ -8,163 +8,128 @@ image:
-
-{% include image.html url="research/roboskin/calibration.png" max-width="75%" description="Illustrated setup of skin units (S) placed along the robots links (L) that are separated by joints (J). We estimate the  Denavit-Hartenberg parameters of each joint in order to calculate the pose of each skin unit along the surface of the robot. " %}
-
-Our optimization algorithm is described as follows.
-
-### 1. Formulate a Kinematic Chain with random values.
+To automatically locate skin units along the surface of a robot, we the angular velocity and linear acceleration measurements from the IMUs. Our optimization algorithm estimates the pose (position and orientation) of a SU using *modified* Denavit-Hartenberg (DH) [1] parameters as illustrated in Figure 2. The pose of each SU is estimated by six DH parameters with respect to the previous joint in the kinematic chain: four parameters from the joint to a virtual joint, and two additional parameters from the virtual joint to the skin unit. **[AR: it is unclear what a virtual joint is---please add more info]**{:.color-banner}
 
-We first represent each skin unit coodinate using a transformation matrix $${}^0T_{SU_i}$$.
+{% include image.html url="research/roboskin/calibration.png" max-width="75%" description="<b>Figure 2.</b> Depiction of multiple skin units (S) placed along the robots links (L) that are separated by joints (J). We estimate the  Denavit-Hartenberg parameters of each joint in order to calculate the pose of each skin unit along the surface of the robot. " %}
 
-$$
-{}^0T_{SU_i} = {}^0T_1 \cdot {}^1T_2 \dots {}^{i-1}T_i \cdot {}^i T_{SU_i} \quad \forall i
-$$
+Our optimization algorithm is composed of the following four steps.
 
-<br/>
+ 1. _Initialize a Kinematic Chain with randomized values._
+    We represent each skin unit coordinate using a transformation matrix
+    $${}^0T_{SU_i} = {}^0T_1 \cdot {}^1T_2 \dots {}^{i-1}T_i \cdot {}^i T_{SU_i}, \quad \forall i$$, where
+    $$
+    {}^{i}T_{i+1} =
+        \left[ \begin{array}{c|c}
+            {}^{i}R_{i+1} & {}^{i}P_{i+1} \\
+            \hline
+            \mathbf{0} & \mathbf{1} \\
+        \end{array}\right].
+    $$
+    **[AR: I don't think the rotation part of this is correct---I know we've been through this already but it needs revision. I just removed it for the sake of compactness]**{:.color-banner}
```

Comment:
Could you give me a reason why this notation is wrong?

Reply:
Yes! But maybe we should take this offline.

Reply:
@watakandai did y'all end up resolving this? Can you remove the part of the equation that we don't use?